tests/test_CLI.py

```diff
@@ -0,0 +1,44 @@
+import os
+import shutil
+
+
+def test_yaml2sbml_cli(script_runner):
+    """Test the command line command `yaml2sbml`."""
+    path = os.path.dirname(os.path.abspath(__file__))
+
+    yaml_dir = os.path.join(path, 'test_yaml2sbml/ode_input1.yaml')
+    sbml_dir = os.path.join(path, 'test_sbml.xml')
+
+    script_runner.run('yaml2sbml', yaml_dir, sbml_dir)
+
+    os.remove(sbml_dir)
```

Comment:
kinda trivial, but maybe compare that the output is the same as one obtained via the python API?

Reply:
good idea, included something like this.